components/ai_chat/resources/untrusted_conversation_frame/components/conversation_entries/index.tsx

```diff
@@ -54,6 +55,14 @@ function ConversationEntries() {
     return event?.completionEvent?.completion ?? ''
   }
 
+  function getTextBetweenTags(text: string, tag: string): string {
```

Comment:
Hey, this function should be lifted outside the component - it doesn't depend on the components internal state so we're just doing extra work